### PR TITLE
chore: Move VueCore to regular Vue

### PR DESCRIPTION
**Pull request by salisdemos**
Since the Bus is gone we now can simplify our Vue setup.
So I move the Stuff from VueCore to the initVue file and now we can use a regular Vue in our config.

This can be finished when this two are merged:
- https://github.com/demos-europe/demosplan-core/pull/1166
- https://github.com/demos-europe/demosplan-core/pull/1165

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/Resources/client/scss/objects/_tooltip.scss [hunk 1]
```diff
@@ -50,7 +50,7 @@ $tooltip-width: 400px;
 //  Tooltip styles used by the Js and Vue implementation
 //  Templates:
 //  * DemosPlanCoreBundle/Resources/client/js/lib/Tooltips.js
-//  * DemosPlanCoreBundle/Resources/client/js/VueConfigCore.js
+//  * DemosPlanCoreBundle/Resources/client/js/InitVue.js
```
Comment (spiess-demos) [inline demosplan/DemosPlanCoreBundle/Resources/client/scss/objects/_tooltip.scss:53]: Templates are located within `client/js/lib/core/libs/Tooltips.js:41` and `@demos-europe/demosplan-ui/src/directives/Tooltip/Tooltip.js:15`

File: client/fe/webpack/plugins.js [hunk 2]
```diff
@@ -100,7 +100,7 @@ const webpackDefaultPlugins = [
 
   // Provide configured Vue and Bus instances to all modules
```
Comment (spiess-demos) [inline client/fe/webpack/plugins.js:101]: This is not correct anymore... 

```suggestion
  // Provide Vue instance to all modules (is configured in InitVue.js before initialization).
```

File: client/js/plugins/loadSentry.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+/**
```
[the remark this answers is not in the document]
Comment (hwiem) [reply client/js/plugins/loadSentry.js:1]: Maybe we could move it to the `js` root directory?

[the remark this answers is not in the document]
Comment (salisdemos) [reply client/js/plugins/loadSentry.js]: moved in

File: client/fe/webpack/plugins.js (hunk 2 above)
Comment (salisdemos) [reply client/fe/webpack/plugins.js]: changed in 6852b8a27abb59355b169831fc0863f1736e9ab6

File: demosplan/DemosPlanCoreBundle/Resources/client/scss/objects/_tooltip.scss (hunk 1 above)
Comment (salisdemos) [reply demosplan/DemosPlanCoreBundle/Resources/client/scss/objects/_tooltip.scss]: changed in 6852b8a27abb59355b169831fc0863f1736e9ab6

**Review by spiess-demos — APPROVED** [review]
lgtm! (code review)

**Pull request merged**